client/src/components/review.jsx

```diff
@@ -4,47 +4,41 @@ import ReviewText from './reviewText';
 import Report from './report';
 import HostResponse from './hostResponse';
 
-class Review extends React.Component {
-  render() {
-    const month = {
-      '01': 'January',
-      '02': 'February',
-      '03': 'March',
-      '04': 'April',
-      '05': 'May',
-      '06': 'June',
-      '07': 'July',
-      '08': 'August',
-      '09': 'September',
-      '10': 'October',
-      '11': 'November',
-      '12': 'December',
-    };
-    const {
-      review,
-    } = this.props;
+const Review = ({ review }) => {
+  const month = {
+    '01': 'January',
+    '02': 'February',
+    '03': 'March',
+    '04': 'April',
+    '05': 'May',
+    '06': 'June',
+    '07': 'July',
+    '08': 'August',
+    '09': 'September',
+    '10': 'October',
+    '11': 'November',
+    '12': 'December',
```

Comment:
Delete last comma.

Reply:
How can I tell if I'm using the write eslint? The eslint I'm using told me to put the comma there, but you and Brian have told me not to. I thought I was using Airbnb...

Reply:
Oh, this was just an eye thing. You are correct. Airbnb style guide wants us to use the horrid dangling comma for multiline object literals.